### PR TITLE
New poem on the Git Tutorial Sandbox README

### LINKED ISSUE #4: Add a second poem to the README
README needs a new poem

**What is required**
Add a poem to the readme.

**Subtasks:**
* [ ] Add subtasks or sections of content if applicable
* [ ] Identify documentation that should be updated upon completion

**How do you know it has to be done**
Link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc. explaining why you think this should be done.

Link to requirements or risk analysis if applicable.

**Starting points**
Where might one start with solving this problem? Point to any relevant files or websites. Even if you have but a vague idea, please share. Perhaps something has prevented you or blocked you from solving the problem already?

**Don't forget (can delete these when done)**
* Put the issue in a milestone
* Assign it to one of the projects
* Add appropriate labels
* Mention dependent/blocked issues if any

### PR DESCRIPTION
**Pull request by edwinRespira**
# Description

Summary of changes

Relevant motivation and context

Dependencies that are required for this change

Closes #4

## Self-review checklist:

- [x] Self-review: looked through the `Files changed` tab, browsed repository in branch
- [x] Documentation updated - reflects changes in code, electrical or mechanical design
- [x] Documentation and graphics follow the [documentation style guide](https://github.com/RespiraWorks/Ventilator/wiki/Documentation-style-guide)
- [x] New content is linked, easily discoverable, does not require too many clicks
- [x] Follows other relevant parts of [contributor wiki](https://github.com/RespiraWorks/Ventilator/wiki)
- [x] PR has a descriptive name
- [x] Tagged relevant reviewers

### Software only:

- [ ] Follows our [code style](https://github.com/RespiraWorks/Ventilator/wiki/Code-style)
- [ ] Commented code, particularly in hard-to-understand areas
- [ ] All tests pass - new and old, locally and on CI
- [ ] No new warnings or static check failures
- [ ] Tests that prove fix is effective or new feature works
- [ ] Manual tests are explained, with instructions for reproducing them

### CAD / mechanical only:

- [ ] Follows our [mechanical engineering standards](https://github.com/RespiraWorks/Ventilator/wiki/Mechanical-engineering-standards)
- [ ] Everything is metric and ISO where possible
- [ ] All features and constraints are named
- [ ] Imported parts are well-named, with source URL and iProperties
- [ ] Screenshots and/or assembly photos updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,59 @@
 Practice repo for people to try things and learn to use git and github
 
 Here I made a change that will cause another branch to be out of date for the PR
+
+Here's a nice poem:
+
+```
+Romans in Dorset: A.D. MDCCCXCV
+BY LOUISE IMOGEN GUINEY
+A stupor on the heath,
```
Comment (martukas) [inline README.md:12]: Maybe leave one blank line before the poem starts?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,59 @@
...
+
+            An instant: gusts again,
+            Then heaven’s impacted wall,
+            The hot insistent rain,
+The thunder-shock; and of the Past mirage no more at all,
+
+            No more the alien dream
+            Pursuing, as we went,
+            With glory’s cursèd gleam:
+Nor sin of Cæsar’s ruined line engulfed us, innocent.
```
Comment (martukas) [inline README.md]:
```suggestion
Nor sin of Caesar’s ruined line engulfed us, innocent.
```

**Review by martukas — CHANGES_REQUESTED** [review]
Pretty good but could be better....

File: README.md (hunk 1 above)
Comment (edwinRespira) [reply README.md:12]: hey, thanks, this has been added

**Review by martukas — APPROVED** [review]
LGTM

**Pull request merged**